test/metabase/api/field_test.clj

```diff
@@ -744,11 +745,9 @@
           (is (seq (nested-fields))))
         (testing "nested fields are removed when json unfolding is disabled"
           (enable-json-unfolding! false)
-          (sync/sync-database! db)
           (is (empty? (nested-fields))))
         (testing "nested fields are added when json unfolding is enabled again"
           (enable-json-unfolding! true)
-          (sync/sync-database! db)
```

Comment:
Here, sync is not necessary because the fields are automatically updated to have active=true.